<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbOwnModelSetComponent</class>
 <widget class="QFrame" name="CDbOwnModelSetComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="vl_DbOwnModelSetComponent">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="swift::gui::views::CAircraftModelView" name="tvp_OwnModelSet">
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fr_OwnModelSetBottom">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="hl_OwnModelSetButtons">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="swift::gui::components::CSimulatorSelector" name="comp_SimulatorSelector">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_OwnModelSetButtons">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_Simulator">
        <property name="text">
         <string>Simulator:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_Simulator">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Simulator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_ShowStatistics">
        <property name="text">
         <string>statistics</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_Clear">
        <property name="text">
         <string>clear set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_CreateNewSet">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>create a new set</string>
        </property>
        <property name="text">
         <string>new set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_FirstSet">
        <property name="text">
         <string>first set</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_CopyFromAnotherSwift">
        <property name="text">
         <string>copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_LoadExistingSet">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>load an existing set</string>
        </property>
        <property name="text">
         <string>load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_SaveAsSetForSimulator">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CAircraftModelView</class>
   <extends>QTableView</extends>
   <header>gui/views/aircraftmodelview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>gui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tvp_OwnModelSet</tabstop>
  <tabstop>le_Simulator</tabstop>
  <tabstop>pb_ShowStatistics</tabstop>
  <tabstop>pb_Clear</tabstop>
  <tabstop>pb_CreateNewSet</tabstop>
  <tabstop>pb_FirstSet</tabstop>
  <tabstop>pb_CopyFromAnotherSwift</tabstop>
  <tabstop>pb_LoadExistingSet</tabstop>
  <tabstop>pb_SaveAsSetForSimulator</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
